Project.toml

```
@@ -1,7 +1,7 @@
 name = "GradedArrays"
 uuid = "bc96ca6e-b7c8-4bb6-888e-c93f838762c2"
 authors = ["ITensor developers <support@itensor.org> and contributors"]
-version = "0.2.4"
+version = "0.2.5"
```

```suggestion
version = "0.3.0"
```
since this is technically breaking.